extensions/mutiny/deployment/src/main/java/io/quarkus/mutiny/deployment/MutinyProcessor.java

```diff
@@ -2,26 +2,21 @@
 
 import java.util.concurrent.ExecutorService;
 
-import io.quarkus.deployment.Feature;
 import io.quarkus.deployment.annotations.BuildStep;
 import io.quarkus.deployment.annotations.ExecutionTime;
 import io.quarkus.deployment.annotations.Record;
 import io.quarkus.deployment.builditem.ExecutorBuildItem;
-import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.ShutdownContextBuildItem;
 import io.quarkus.mutiny.runtime.MutinyInfrastructure;
 
 public class MutinyProcessor {
 
-    @BuildStep
-    public FeatureBuildItem registerFeature() {
```

Comment:
@cescoffier I have added this as a dep to vertx-http, so the correct executor is always setup, but given that this basically just sets up some infrastructure IMHO this extension is basically just an implementation detail, so should not be listed as a feature, WDYT?

Without adding this as a dep the dev mode tests leak all their ClassLoaders, as the default executors created by Mutiny prevent the CL from being freed.

Reply:
Not being listed is not a problem. With Resteasy Reactive always bringing it, it's often there (the fact it was listed was important when it was something to be added, and so easier for me to detect the missing feature).

About the dependency, no problem for me. 
In addition, @mkouba is looking at reorganizing the Vert.x extensions and would make the dependency directly into the vert.x core extension (and so expose the Mutiny variant of vert.x every time). The idea is to merge the vert.x extension and the vert.x core extension (mutiny is the only "extra Vert.x"  we support in 2.x, and merge the `@RouteFilter` into the `vertx-http` extension.